website/docs/DriverBinaries.md

```diff
@@ -3,7 +3,79 @@ id: driverbinaries
 title: Driver Binaries
 ---
 
-To run automation based on the WebDriver protocol you need to have browser drivers set up that translate the automation commands and are able execute them in the browser. With WebdriverIO `v8.14` and above there is no need to manually download and setup any browser drivers anymore as this is handled by WebdriverIO. You only need to have a browser installed, that's it!
+To run automation based on the WebDriver protocol you need to have browser drivers set up that translate the automation commands and are able to execute them in the browser.
+
+## Automated setup
+
+With WebdriverIO `v8.14` and above there is no need to manually download and setup any browser drivers anymore as this is handled by WebdriverIO. All you have to do is specify the browser you want to test and WebdriverIO will do the rest. For more information on this feature, see [here](capabilities#automate-different-browser-channels).
+
+### Customizing the level of automation
+
+WebdriverIO's have three levels of automation:
+
+**1. Download and install the browser using [@puppeteer/browsers](https://www.npmjs.com/package/@puppeteer/browsers).**
+
+WebdriverIO will only do this if it cannot autodetect an installation of the browser with the `browserName`/`browserVersion` combination specified in the [capabilities](configuration#capabilities-1) configuration. If `browserVersion` is omitted, an arbitrary version will be used instead.
```

Comment:
If `browserVersion` is omitted WebdriverIO will try to use the browser installed on the OS and if not found will install the current stable browser release.

Reply:
https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/packages/wdio-utils/src/driver/utils.ts#L16

https://github.com/webdriverio/webdriverio/blob/aa35c291fb496b80350766d674385ea7b4d1516a/packages/wdio-utils/src/driver/utils.ts#L117-L133

Not the best choice of words. I used "arbitrary" as `locate-app` will use whatever is on the `PATH` env var (at least on linux) ([ref1], [ref2], [ref3]).

[ref1]: https://github.com/hejny/locate-app/blob/978634416b7c039c55c5c4db7a7aac868f6c6a9b/src/browsers/locateChrome.ts
[ref2]: https://github.com/hejny/locate-app/blob/978634416b7c039c55c5c4db7a7aac868f6c6a9b/src/locateApp.ts
[ref3]: https://github.com/hejny/locate-app/blob/978634416b7c039c55c5c4db7a7aac868f6c6a9b/src/platforms/locateAppOnLinux.ts

Reply:
How about we use a mixed version of both. I just want to make sure that we tell people that WebdriverIO looks for a browser version on the OS. You are right that this installed version might be arbitrary if the user has multiple versions installed.